.travis.yml

```diff
@@ -12,4 +12,5 @@ install:
   - export PATH=$(brew --prefix llvm)/bin:$PATH
 
 script:
+  - export CODEBUILD_SRC_DIR=$(pwd)
```

Comment:
alternatively, common-posix.sh could do this instead of using CODEBUILD_SRC_DIR:
```
SRC_DIR=`pwd`
```

Reply:
No it can't, because everything has to be based on the codebuild env vars, or it's not guaranteed to work. The working directory has shifted on us.

Reply:
🥇